### PR TITLE
router: request body example

### PR DESCRIPTION
**Pull request by arsandev**
I just confirmed a few writing bugs in the example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -471,6 +471,9 @@ router
     if (context.params && context.params.id && books.has(context.params.id)) {
       context.response.body = books.get(context.params.id);
     }
+  })
+  .post("/", async (context) => {
```
Comment (kitsonk) [inline README.md:475]: Why are you adding code here?

File: examples/routingServer.ts [hunk 2]
```diff
@@ -46,6 +46,9 @@ router
     } else {
       return notFound(context);
     }
+  })
+  .post("/", async (context) => {
```
Comment (kitsonk) [inline examples/routingServer.ts:50]: Why are you adding code here?

File: README.md (hunk 1 above)
Comment (arsandev) [reply README.md:475]: I want to make sure people know how to write multiple methods and handle requests

File: examples/routingServer.ts (hunk 2 above)
Comment (arsandev) [reply examples/routingServer.ts:50]: I want to make sure people know how to write multiple methods and handle requests

**Comment by kitsonk** [comment]
Sorry, we needed to provide a better example, and something that worked like a way it was expected.  I have added something logical to the `example/routingServer.ts` and I don't believe the rest of the documentation needs to give examples specifically using other HTTP methods, as those are well documented in the API.

**Pull request closed without merging**